### ISSUE TITLE
/users/login route broken when accessed via curl

### ISSUE THREAD

**Issue by mattsouth** [meteor-compat/meteor-rest#2]
The rest route ``/users/login`` from rest-accounts-password now fails when accessed via curl.  To replicate this issue you can clone the project and run the test with ``meteor test-packages ./packages/rest-accounts-password``.  The tests pass and set up a web app that  you can test with curl.  You can successfully use the /users/register route, i.e.

```
curl http://localhost:3000/users/register -d "username=test1&password=test1"
{
  "token": "zNPRB8c72B8uNwdOo-8vQQ-xEPd21vq5oSDSkJJUl5q",
  "tokenExpires": "2023-01-18T09:27:33.701Z",
  "id": "CsXQfo4HFSNBAHwgS"
}
```

however if you try /users/login then you will see a server error

```
curl http://localhost:3000/users/login -d "username=test1&password=test1"
{
  "error": "internal-server-error",
  "reason": "Internal server error"
}
``` 

which wasnt there in previous versions.  Im pretty sure it fails on the options.hasOwnProperty call here: https://github.com/meteor-compat/meteor-rest/blob/3f86a467c6d86f790f4d5eeea1e4a33bf6e5d4f3/packages/rest-accounts-password/rest-login.js#L16 

because the passed req data parameter object doesnt have a prototype and therefore doesnt have the hasOwnProperty function.

**Comment by Grubba27** [comment]
Hey @mattsouth loved that you have created this issue.
You think that making that snippet into: 

```js
 if (options?.hasOwnProperty('email')) { 
```
would solve ?

**Comment by mattsouth** [comment]
That doesnt solve the problem but you can fix it if you change that line to:
```
     if (options.email) {
```
though I wonder if it might be wise to be more defensive than that.

**Comment by filipenevola** [comment]
Published in v1.2.2
